app/models/concept.ts

```diff
@@ -13,8 +14,9 @@ export type BlockJSON = {
 };
 
 export default class ConceptModel extends Model {
-  @belongsTo('user', { async: false, inverse: null }) declare author: UserModel;
+  @belongsTo('user', { async: true, inverse: null }) declare author: UserModel;
```

Comment:
@libmartinito we don't use `async: true` relationships as a rule since they can fire requests at unexpected times. Can you elaborate on why we need this?

Reply:
seems to be fixed. don't need anymore